client/src/components/Sidebar/SidebarHead/SidebarButton/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { SIDEBAR_ITEM } from '../../'
+
+const SidebarButton = ({ value, current, setValue, content }: { value: SIDEBAR_ITEM | null, current: SIDEBAR_ITEM | null, setValue: (arg0: SIDEBAR_ITEM | null) => void, content: any }) => {
+  const selected = current === value;
+  return (
+    <button
+      onClick={() => {
+        if (selected) setValue(null)
```

Comment:
Could you add a comment to describe what this is doing? Making a separate function for it might be nice too (optional)